### PR TITLE
MSPSDS-1147 Move error summaries and notifications banner above h1

### PR DESCRIPTION
**Pull request by jasiekmiko**
Also give them full width

### CODE REVIEW

File: govuk/login/template.ftl [hunk 1]
```diff
@@ -108,83 +108,57 @@
...
+                            </svg>
+                        </#if>
+                        <#if message.type = 'info'>
+                            <svg class="hmcts-banner__icon" fill="currentColor" role="presentation" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" height="25" width="25">
+                                <path d="M13.7,18.5h-2.4v-2.4h2.4V18.5z M12.5,13.7c-0.7,0-1.2-0.5-1.2-1.2V7.7c0-0.7,0.5-1.2,1.2-1.2s1.2,0.5,1.2,1.2v4.8
+C13.7,13.2,13.2,13.7,12.5,13.7z M12.5,0.5c-6.6,0-12,5.4-12,12s5.4,12,12,12s12-5.4,12-12S19.1,0.5,12.5,0.5z" />
+                            </svg>
+                        </#if>
+                        <div class="hmcts-banner__message">
+                            <span class="hmcts-banner__assistive">Success</span>
```
Comment (broder) [inline govuk/login/template.ftl]: I realise that you've only moved this up but do we always want to have the "Success" string here?

Comment (jasiekmiko) [reply govuk/login/template.ftl]: This is for non-visual tools only:
https://github.com/UKGovernmentBEIS/keycloak-theme-govuk/blob/0149fd057c12669236bd7510ca34b56dc5be64d8/sass/banner.scss#L45

Doesn't actually affect the displayed message:
image-2019-03-28-14-10-37-344

Comment (broder) [reply govuk/login/template.ftl]: Yes, but presumably it should say "Success", "Warning" or "Information" depending on the type of message.

Comment (jasiekmiko) [reply govuk/login/template.ftl]: Oops, sorry, you're completely right! Gah, trigger happy me 🤦‍♂️ I'll fix that in another pr :)

**Pull request merged**